ParchePartnerURLScheme.podspec

```diff
@@ -0,0 +1,18 @@
+Pod::Spec.new do |s|
+  s.name         = "ParchePartnerURLScheme"
+  s.version      = '1.0.0'
+  s.summary      = "Parche Partner URL Scheme Helper"
+  s.description  = <<-DESC
+                    A utility to make it easier for partner applications of Parche to open the app using a URL scheme.
```

Comment:
It's [Ruby's here-document syntax](http://en.wikipedia.org/wiki/Here_document#Ruby) for a string literal.  I'd say it's more about being able to use quotes and whatever else without having to escape them plus simple multiline abilities.